### PR TITLE
[bug](backup) Add table/partition id into convert_rowset_ids()

### PR DESCRIPTION
**Pull request by xy720**
## Proposed changes

Issue Number: step 1 in version 1.2 in #37076

### CODE REVIEW

**Comment by xy720** [comment]
run buildall

**Comment by xy720** [comment]
run buildall

File: gensrc/thrift/AgentService.thrift [hunk 1]
```diff
@@ -210,6 +210,8 @@ struct TCloneReq {
     9: optional i64 dest_path_hash;
     10: optional i32 timeout_s;
     11: optional Types.TReplicaId replica_id = 0
+    12: optional i64 table_id
```
Comment (cambyzju) [inline gensrc/thrift/AgentService.thrift:213]: 12 for partition_id, 13 for table_id, keep same order as master and 2.0

Comment (xy720) [reply gensrc/thrift/AgentService.thrift]: done

**Review by cambyzju — APPROVED** [review]

**Review by lide-reed — APPROVED** [review]
LGTM

**Pull request merged**